fwb/backend/supabaseContext.tsx

```diff
@@ -0,0 +1,32 @@
+import exp from "constants";
+import React from "react";
+import { supabaseClient } from "./supabaseClient";
+import { SupabaseClient } from "@supabase/supabase-js";
+import { useAuth } from "@clerk/nextjs";
+
+const SupabaseContext = React.createContext(null);
+
+function SupabaseProvider({ children }: any) {
+  const [supabase, setSupabase] = React.useState<any>(null);
+  const { getToken } = useAuth();
+
+  React.useEffect(() => {
```

Comment:
I would import useEffect and just call the hook rather than using React.useEffect. To stay consistent within our codebase.